string class_id
int32 id 

int32 bounding_box_min_x
int32 bounding_box_min_y
int32 bounding_box_max_x
int32 bounding_box_max_y

sensor_msgs/PointCloud2 pointcloud_in_BB_2D
sensor_msgs/PointCloud2 pointcloud_in_BB_3D

int32[] on_object_point_indices

geometry_msgs/PoseWithCovariance pose


